### PR TITLE
TELCODOCS-1849: Add commands for webhook logs

### PR DESCRIPTION
**Pull request by StephenJamesSmith**
D/S Docs: [MGMT-16640] Move webhooks to a separated deployment. Add commands for webhook logs.

Jira: https://issues.redhat.com/browse/TELCODOCS-1849

Version(s): openshift-4.16.0, openshift-4.15.0, openshift-4.14.0, openshift-4.13.0, openshift-4.12.0

Link to docs preview: https://76309--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hardware_enablement/kmm-kernel-module-management.html#kmm-reading-operator-logs_kernel-module-management-operator

SME review: @qbarrand
QE review: @cdvultur

### CODE REVIEW

**Comment by StephenJamesSmith** [comment]
@cdvultur @qbarrand Please review and ACK.

File: modules/kmm-reading-operator-logs.adoc [hunk 1]
```diff
@@ -8,16 +8,31 @@
...
 ----
 $ oc logs -fn openshift-kmm deployments/kmm-operator-controller
 ----
 
-.Example command for KMM-Hub
+.Example command for KMM webhook server
+
+[source,terminal]
+----
+$ oc logs -fn "$namespace" deployments/kmm-operator-webhook-server`
```
Comment (qbarrand) [inline modules/kmm-reading-operator-logs.adoc:23]:
```suggestion
$ oc logs -fn openshift-kmm deployments/kmm-operator-webhook-server
```

File: modules/kmm-reading-operator-logs.adoc [hunk 2]
```diff
@@ -8,16 +8,31 @@
...
 [source,terminal]
 ----
 $ oc logs -fn openshift-kmm-hub deployments/kmm-operator-hub-controller
 ----
+
+.Example command for KMM-Hub webhook server
+
+[source,terminal]
+----
+$ oc logs -fn "$namespace" deployments/kmm-operator-hub-webhook-server`
```
Comment (qbarrand) [inline modules/kmm-reading-operator-logs.adoc:37]:
```suggestion
$ oc logs -fn openshift-kmm deployments/kmm-operator-hub-webhook-server
```

File: modules/kmm-reading-operator-logs.adoc (hunk 1 above)
Comment (StephenJamesSmith) [reply modules/kmm-reading-operator-logs.adoc:23]: changed.

File: modules/kmm-reading-operator-logs.adoc (hunk 2 above)
Comment (StephenJamesSmith) [reply modules/kmm-reading-operator-logs.adoc:37]: changed.

File: modules/kmm-reading-operator-logs.adoc [hunk 3]
```diff
@@ -8,16 +8,30 @@
...
 ----
 $ oc logs -fn openshift-kmm deployments/kmm-operator-controller
 ----
 
-.Example command for KMM-Hub
+.Example command for KMM webhook server
+
+[source,terminal]
+----
+$ oc logs -fn openshift-kmm deployments/kmm-operator-webhook-server`
```
Comment (cdvultur) [inline modules/kmm-reading-operator-logs.adoc:22]: Please remove extra quote ( ` ) at the end of the command

File: modules/kmm-reading-operator-logs.adoc [hunk 4]
```diff
@@ -8,16 +8,30 @@
...
 [source,terminal]
 ----
 $ oc logs -fn openshift-kmm-hub deployments/kmm-operator-hub-controller
 ----
+
+.Example command for KMM-Hub webhook server
+
+[source,terminal]
+----
+$ oc logs -fn openshift-kmm deployments/kmm-operator-hub-webhook-server`
```
Comment (cdvultur) [inline modules/kmm-reading-operator-logs.adoc:36]: Please remove extra quote ( ` ) at the end of the command

**Review by cdvultur — CHANGES_REQUESTED** [review]
Some extra quotes are present, please remove them

**Review by cdvultur — APPROVED** [review]
/lgtm

**Review by skrthomas — COMMENTED** [review]
/lgtm

**Pull request merged**